SI/Sem.07/Pract.07/Strings/Task04.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+using namespace std;
+
+void replaceSymbols(char* arr, char x, char a, char b)

```

Comment:
Parameters should be named better

Reply:
@PresiyanaBB could you address the naming comment?